src/lib/ngsi/ContextElementResponse.cpp

```diff
@@ -134,6 +134,7 @@ ContextElementResponse::ContextElementResponse
 (
   const mongo::BSONObj&  entityDoc,
   const AttributeList&   attrL,
+  AttributeList*         metadataList,
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/2590/commits/126e9506763c2389da6cac16d89e15da2a271fba